### PR TITLE
Fix event creation issue with white spaces.

### LINKED ISSUE #1021: Able to create events with white spaces only.
**Describe the bug**
When attempting to create an event using the event creation modal, it is possible to submit the form with no title, no description, and no location, by simply inputting spaces.

**To Reproduce**

1. Login to talawa admin.
2. Go to the dashboard and then events.
3. Click on "Add event" button.
4. Enter spaces in the title field.
5. Enter spaces in the description field.
6. Enter spaces in the location field.
7. Attempt to create the event with the filled but empty information.

**Expected behavior**
I expected the system to validate and prevent the creation of an event with no meaningful information, ensuring at least one of the following fields (title, description, location) is non-empty.

**Actual behavior**
The system allows the creation of an event even when all fields (title, description, location) contain only white spaces.

**Screenshots**

https://github.com/PalisadoesFoundation/talawa-admin/assets/109683163/3daa1b17-d65d-487b-9fba-988f93f85028

**Additional details**

### PR DESCRIPTION
**Pull request by skbhagat0502**
**What kind of change does this PR introduce?**

A bug.

**Issue Number:**

Fixes #1021 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa-admin/assets/109683163/edc731eb-d87f-480b-85e4-3631cba75bf0

**If relevant, did you update the documentation?**

**Summary**

Previously events can be created with white spaces only. To fix this issue I have added a validation that if the title, description or the location has values with white spaces only than It will show a warning. I have attached a video for the same showing that.
Now user can't create an event with white spaces and no text.

**Does this PR introduce a breaking change?**

No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
Please fix the failing tests. The files that you edited only have 57% code coverage

**Comment by skbhagat0502** [comment]
Ok @palisadoes

File: src/screens/OrganizationEvents/OrganizationEvents.tsx [hunk 1]
```diff
@@ -83,41 +83,53 @@ function organizationEvents(): JSX.Element {
...
+        }
+      } catch (error: any) {
+        /* istanbul ignore next */
+        errorHandler(t, error);
       }
-    } catch (error: any) {
-      /* istanbul ignore next */
-      errorHandler(t, error);
+    } else if (formState.title.trim().length === 0) {
+      toast.warning('Please enter a valid title!');
```
Comment (EshaanAgg) [inline src/screens/OrganizationEvents/OrganizationEvents.tsx:128]: Remove these error messages with more descriptive one's that tell the user that the same can't be blank.

**Review by EshaanAgg — CHANGES_REQUESTED** [review]
Please add tests for the proposed changes.

**Comment by skbhagat0502** [comment]
> Please add tests for the proposed changes.

Yes I am working on it.

File: src/screens/OrganizationEvents/OrganizationEvents.tsx (hunk 1 above)
Comment (skbhagat0502) [reply src/screens/OrganizationEvents/OrganizationEvents.tsx]: I changed the error messages. as `Title can not be blank`, `Description can not be blank` and `Location can not be blank`

**Comment by skbhagat0502** [comment]
@EshaanAgg There are no more lines of code to write test for in `OrganisationEvents.tsx`. The test coverage percentage is less due to other components.

**Comment by skbhagat0502** [comment]
@EshaanAgg @Ayush0Chaudhary Please review.

**Review by EshaanAgg — APPROVED** [review]
LGTM! Thanks.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**